client-src/elements/chromedash-guide-stage-page.js

```diff
@@ -162,7 +162,9 @@ export class ChromedashGuideStagePage extends LitElement {
       const hash = decodeURIComponent(location.hash);
       if (hash) {
         const el = this.shadowRoot.querySelector(hash);
-        el.scrollIntoView(true, {behavior: 'smooth'});
+        if (el) {
+          this.shadowRoot.querySelector(`chromedash-form-field[name="${el.name}"] tr th b`).scrollIntoView(true, {behavior: 'smooth'});
```

Comment:
Do we need the `tr th b`?  What if it just scrolled into view the custom chromedash-form-field element that has name="..."?

Reply:
ya i tried that, `this.shadowRoot.querySelector(chromedash-form-field[name="${el.name}"]).scrollIntoView(true, {behavior: 'smooth'})`, but a gap is coming on top of the label
Sample images

![image](https://user-images.githubusercontent.com/62694340/212458566-817f29f9-7e3e-4f39-b92c-bd4517d80553.png)
![image](https://user-images.githubusercontent.com/62694340/212458582-fd621de8-939a-4726-bf25-2ba1b7cd1d1d.png)